apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -203,17 +203,25 @@ class TopInstructions extends Component {
   }
 
   /**
-   * Function to force the height of the instructions area to be the
+   * Function to force the height of the instructions area when the comment tab is chosen to be the
    * full size of the content for that area. This is used when the comment
    * tab loads in order to make the instructions area show the whole
    * contents of the comment tab.
    */
-  forceTabResizeToMaxHeight = () => {
+  forceCommentTabResizeToMaxHeight = () => {
     if (this.state.tabSelected === TabType.COMMENTS) {
```

Comment:
i'm sorry, i miscommunicated my initial intention for this refactor. i meant for this to remove the guard on this line, so that, rather than checking if we're on the comments tab in the method, we're only calling the method if we're sure we want to force the tab resize. does that make sense?